ReleaseProcessAutomation.Tests/SemanticVersioning/SemanticVersionTests.cs

```diff
@@ -46,6 +46,25 @@ public void GetNextPossibleVersionsDevelop_WithPreReleaseAlpha_ReturnsSixVersion
     Assert.That(possibleVersions, Does.Contain(parser.ParseVersion("1.3.5-beta.1")));
   }
 
+  [Test]
+  public void GetNextPossibleVersionsDevelop_WithPreReleaseRC_ReturnsProperVersions()
```

Comment:
The test-name refers to develop-branch. the RC-version should only be on release-branches. When you're on develop, you get the info-text that you need to make a release-branch for RC versions. 

This seems to also affect the helper-method `GetNextPossibleVersionsDevelop`.